pdns/lua-record.cc

```diff
@@ -97,15 +97,21 @@ class IsUpOracle
       if (cd.opts.count("useragent")) {
         useragent = cd.opts.at("useragent");
       }
-      MiniCurl mc(useragent);
+      int http_code = 200;
+      if (cd.opts.count("httpcode")) {
+        http_code = pdns_stou(cd.opts.at("httpcode").c_str());
```

Comment:
You can lose `c_str()` I believe - `pdns_stou` takes string. I see now that I did not mention that earlier, I just thought I did!

Reply:
I made the modification, and also re-made some tests which are OK.